common/OpTestThread.py

```diff
@@ -73,16 +74,21 @@ def inband_child_thread(self, threadName, cmd_list, sleep_time, torture_time, ig
             for cmd in cmd_list:
                 if ignore_fail:
                     try:
-                        self.c.run_command(cmd)
+                        self.c.run_command(cmd, timeout=120)
```

Comment:
is this valid on large config as well

Reply:
can we have wrapper around like it can be controlled by testcase

Reply:
Could you please elaborate?

Reply:
ah @rashijhawar missed this notification, what I was  pointing like

 can have timeout pram in OpSSHThreadLinearVar1 constructor argument and test should able to override that and for and even we can define a default value 
 
 like in the LPM use case 
```
self.thread_stressng = OpSSHThreadLinearVar1(1, 'stressng-thread', [modified_stressng_cmd], 0, thread_timeout, True,120)

```
so we should not hard code and even in the future  if required change the use case should be able to overwrite without touching the utility

Reply:
Done